examples/serverless/lambda-http-ddb-datasources/resolvers/ddb/addTodo.js

```diff
@@ -0,0 +1,34 @@
+import { util } from '@aws-appsync/utils';
+import * as ddb from '@aws-appsync/utils/dynamodb';
+
+export function request(ctx) {
+	// Without ddb util
```

Comment:
I'd just remove this. simpler with the ddb utils

Reply:
Done